### DISCUSSION TITLE
Hiding irrelevant random repos on new feed

### DISCUSSION THREAD

**Discussion by JC3** [community/community#59073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How do I hide "trending repositories", "repositories recommended for you", and the "explore repositories" pane? I'm not interested in random repositories, I'm interested in getting my work done.

**Comment by anhtran917** [comment]
For this issue, you can adjust your GitHub settings and utilize browser extensions such as GitHub Customizer.

**Reply by CrzyMan** [reply]
Hi there. I looked through my settings and couldn't tell which ones you were talking about. Could you explain a bit more?
